### PR TITLE
Add Release Notes for iOS Release 1.62.1

### PR DESCRIPTION
**Pull request by Uni-verse**
## Summary of Changes

This pull request fixes #[8782](https://github.com/brave/brave-ios/issues/8782)

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [1.62.1](https://github.com/brave/brave-ios/releases/tag/v1.62.1)
+
+### Web3
+
+- Fixed Buy/Send/Swap action buttons not working from Asset details page. [(#8776)](https://github.com/brave/brave-ios/issues/8776)
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: maybe add `Asset` in quotes to it's easier to read. Example:

```
- Fixed Buy/Send/Swap action buttons not working from "Asset" details page. [(#8776)](https://github.com/brave/brave-ios/issues/8776)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+### General
+
+- Updated JitsiSDK to version 8.6.3. [(#8777)](https://github.com/brave/brave-ios/issues/8777)
+- Fixed insecure state being shown in address bar falsely for some websites. [(#8778)](https://github.com/brave/brave-ios/issues/8778)
```
Comment (kjozwiak) [inline CHANGELOG.md]: hmm what about the following? I don't think the issue was happening only on certain websites. I believe it was a general issue but we managed to reproduced using certain websites. Maybe something along the lines of the following?

```
- Fixed insecure state being falsely displayed in address bar in certain cases. [(#8778)](https://github.com/brave/brave-ios/issues/8778)
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]
Some small nits/suggestions 👍

**Comment by Uni-verse** [comment]
> Some small nits/suggestions 👍

Thanks! https://github.com/brave/brave-ios/commit/4456b642b92ebbc0a2df9f105a4937760c8cabe2

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**